tests/Microsoft.Identity.Test.LabInfrastructure/LabUserHelper.cs

```diff
@@ -57,11 +57,24 @@ public static async Task<string> GetMSIEnvironmentVariablesAsync(string uri)
             return result;
         }
 
+        /// <summary>
+        /// Returns the AAD cloud user idlab1@msidlab4.onmicrosoft.com
+        /// </summary>
+        /// <returns></returns>
         public static Task<LabResponse> GetDefaultUserAsync()
         {
             return GetLabUserDataAsync(UserQuery.PublicAadUserQuery);
         }
 
+        /// <summary>
+        /// Returns the AAD cloud user idlab2@msidlab4.onmicrosoft.com
```

Comment:
The user referenced in `PublicAadUser2Query` is actually idlab@msidlab4.onmicrosoft.com